packages/http-client/src/http-client.ts

```diff
@@ -142,12 +143,12 @@ export function buildHttpRequestConfigWithOrigin(
 }
 
 /**
- * This method does nothing and is only there to indicate that the call was made by an OData or OpenApi client and encoding is already done on filter and key parameters.
+ * This method does nothing and is only there to indicate that the call was made by an typed OData client and encoding already happened on in the client.
  * @param params - Parameters which are returned.
  * @returns The parameters as they are without encoding.
  * @internal
  */
-export const encodeTypedClientRequest: ParameterEncoder = (
+export const odataTypedClientParameterEncoder: ParameterEncoder = (
```

Comment:
```suggestion
export const oDataTypedClientParameterEncoder: ParameterEncoder = (
```

I noticed in other places in the code, we write oData (e.g. commonODataUri). I would keep the casing consistent. I also noticed this in few other places in the PR